.github/workflows/publish-javadoc.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy Javadoc
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+
+jobs:
+  PublishJavaDoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - shell: bash
+        run: |
+          git remote set-head origin --auto
+          git remote add upstream https://github.com/linkedin/venice
+          git fetch upstream
+          git config --global user.name "JavaDoc Bot"
+          git config --global user.email "javadoc_bot@invalid.com"
```

Comment:
Yes, I had to set this up otherwise the workflow failed as there's no committer identity. See [this failure case.](https://github.com/adamxchen/venice/actions/runs/5060599436/jobs/9083691757).